atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -624,7 +624,13 @@ def run(self):
                 if output.file:
                     output.file.close()
 
-        session.CGImport(koji_metadata, server_dir)
+        build_info = session.CGImport(koji_metadata, server_dir)
+        build_id = build_info["id"]
```

Comment:
Wonder if we should deal with build_info being None here? That will happen when running against a Koji that doesn't have the CGImport() fix. In that case, we should return None as the build_id, but we'll have to make sure we don't try storing a nil value in the annotation because OpenShift will complain.